data-types/fixed-length-ascii/README.md

```diff
@@ -0,0 +1,33 @@
+# Fixed-length ASCII data type
+
+Defines a data type for fixed-length ASCII strings.
+
+## Permitted fill values
+
+The value of the `fill_value` metadata key must be a string.
```

Comment:
Why is the length specified in bits?  What does it mean if you specify a length that is not a multiple of 8?